crates/transaction-pool/benches/reorder.rs

```diff
@@ -88,8 +88,9 @@ fn txpool_reordering_bench<T: BenchTxPool>(
                     txpool.add_transaction(new_tx);
                 }
                 let smaller_base_fee = base_fee.saturating_sub(10);
-                txpool.reorder(smaller_base_fee);
-            })
+                txpool.reorder(smaller_base_fee)
+            };
+            black_box(())
```

Comment:
not sure, please advise @rkrasiuk

Reply:
`black_box` tries to prevent the optimization of whatever is inside. and we can now actually use `std::hint::black_box` instead of the criterion one.

So having it like that, doesn't actually do anything. So either remove it, or add a clippy ignore if we're sure it's actually doing its job

Reply:
feel free to make these changes as you see fit, I'm not familiar with any of this